### ISSUE TITLE
Cisco IOS -TELNET - No Enable template not work

### ISSUE THREAD

**Issue by jnizamj** [rconfig/rconfig#75]
Hi,
I have several Cisco switches access using telnet were unable to capture the config. When I change to SSH templates, it's work as normal. All Cisco elements whichare  using Cisco IOS - TELNET - No Enable template  will return the following output during debug:-
BTW, I've tried to modify the template, but fruitless. Thanks.

[root@dory ~]# php /home/rconfig/lib/showCmdScript.php 530426
The /home/rconfig/lib/showCmdScript.php script was run from a manual invocation on a shell with Task ID:530426
13 February 2018 11:30:58 AM
Array
(
    [id] => 11
    [deviceName] => LSKPA-SW
    [deviceIpAddr] => 10.xx.xx.xx
    [deviceEnablePrompt] => 
    [devicePrompt] => LSKPA-SW#
    [deviceUsername] => xxxx
    [devicePassword] => xxxx
    [deviceEnablePassword] => 
    [templateId] => 1
    [nodeCatId] => 11
)
Success: Connected to LSKPA-SW (10.xx.xx.xx) for taskID 530426 - in (File: /home/rconfig/lib/showCmdScript.php)
13 February 2018 11:31:03 AM
show running-config view full\r\n------------------------------------------------------\r\n13 February 2018 11:31:03 AM
Array
(
)
The owner permisions for directory /home/rconfig/data/ were reset to owner apache because script was run interactively

**Comment by rconfig** [comment]
Hi,

You will need to purchase support for assistance. I can tell this is a device or configuration, or password issue. 

Thanks

**Comment by ayad99** [comment]
Hi

We had the same issue. I have checked all the points suggested here (and in kb2001) by rConfig team. But it simply does not work with Telnet. Interestingly I used another tool and it picked up the config dump fine using the same Telnet credentials. So this is not a device config issue, or device credential issue. And this is not a rConfig issue as the settings are pretty straight forward. SSH works fine though in rConfig. It seems like there is a hidden bug in rConfig.

I understand that the rConfig tool is free, but the support is paid. But the way rConfig team is asking to buy support to identify this hidden issue appears bit unethical. It would have been better if rConfig was not offered as a free tool and rather had a paid subscription in the first place. At least we would all know what to expect.
